dbt/models/churnado_subscriptions.sql

```diff
@@ -0,0 +1,41 @@
+select
+  s.id as subscription_id
+  , s.customer_id
+  , s.plan_id
+  , s.simplified_plan_id
+  , s.canceled_at
+  , s.created_at
+  , s.quantity
+  , s.trial_start_at
+  , s.trial_end_at
+  , u.id as user_id
+  , datediff(day, s.trial_start_at, s.trial_end_at) as trial_length
+  , case -- if subscription was active on Dec 31 and was created two months before, we can use it in the training set
+      when s.created_at < '2017-11-01' and (s.canceled_at is null or s.canceled_at > '2017-12-31')
+      then true
+      else false
+    end as is_training
+  , count(distinct case when c.captured and c.paid and c.refunded = false then c.id else null end) as successful_charges
+  , count(distinct case when c.refunded then c.id else null end) as refunded_charges
+  , count(distinct case when c.paid = false and c.captured = false then c.id else null end) as failed_charges
+from dbt.stripe_subscriptions as s
+inner join dbt.users as u
```

Comment:
I think we can remove this for now and do the joins in the last `train_features.sql` model. Curious if that makes sense?

Reply:
Sure, we could do that. Just out of curiosity, what difference would it make? 😄

Reply:
Not a huge difference but since the final `train_features.sq` will have to be joined anyways, we can reduce joins doing all of them there.